src/types/Account/methods/createTransaction.js

```diff
@@ -77,7 +76,7 @@ function createTransaction(opts) {
     return utxo;
   });
 
-  const feeCategory = (opts.isInstantSend) ? 'instant' : 'normal';
+  const feeCategory = 'normal';
```

Comment:
Do you still need feeCategory?

Reply:
Oh thank for looking at it. 
That's a really great question, and my thinking is yes. 
Fee are today really easy because we do not have that much miner that are annoying (I know only GenesisMining doing that), but if anything changes, and that's the option I think will happen then we might be really picky. 
My point is, while it's not totally done because always tried to do the minumu, we need to allow fee, but I still want to wait that the fee structure to platform happen, so we can also have the whole picture in mind with fee on both L1 and L2 (relative to if we provide fee info to an endpoint)

Reply:
I agree @Alex-Werner. I think it would be premature to remove it now and we should keep it until it is clear it is definitely unnecessary.